.github/actions/node/action.yml

```diff
@@ -8,6 +8,8 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: actions/setup-node@v4
+    - uses: actions/setup-node@1d0ff469b7ec7b3cb9d8673fde0c81c44821de2a # v4.2.0
```

Comment:
I don't think we should do this until that the automated process for updating these hashes are expanded to also check the `.github/actions` directory. I think right now it's just checking the `.github/workflows` repo. I've been meaning to raise this as an issue. Until then, this will be stuck at this hardcoded hash which is probably not what we want

Reply:
Nevermind... https://github.com/DataDog/dd-trace-js/pull/5448 will fix it